src/components/util-browser.js

```diff
@@ -80,7 +80,7 @@ function nextComponentId() {
     // marko runtimes. This allows multiple instances of marko to be
     // loaded in the same window and they should all place nice
     // together
-    return "b" + markoUID.i++;
+    return "c" + markoUID.i++;
```

Comment:
This allows the newest marko version to be compatible with every previous one. However we won't be able to make <4.4 and >4.5 versions compatible because the global object that held the runtimeId value gets deleted: https://github.com/marko-js/marko/blob/0f5e639775cb23a75fea5669e4a651ded537f6f7/src/components/init-components-browser.js#L219